### ISSUE TITLE
Migrate to Openssl ruby standard library

### ISSUE THREAD

**Issue by ChrisLundquist** [titanous/yubikey#6]
The previous crypt gem was yanked as noted: https://github.com/titanous/yubikey/pull/5

**Comment by ChrisLundquist** [comment]
@ukd1 @titanous 

was 

``` ruby
Crypt::Rijndael.new(@aes_key, 128).decrypt_block(@token)
```

# 

``` ruby
decipher = OpenSSL::Cipher.new('AES-128-CBC').decrypt
plain_text = decipher.update(@token) + decipher.final
```

**Comment by ukd1** [comment]
Yep. Just tidying up. Should I stop working on my copy?

**Comment by ChrisLundquist** [comment]
@ukd1 I was just doing that research. It looked like that was the heart of the change. If you have code, the ball is yours.

Thank you!

**Comment by ukd1** [comment]
Yep! I've got to fix this today :-)

**Comment by ChrisLundquist** [comment]
Working on this here, https://github.com/blueboxgroup/yubikey/tree/openssl

**Comment by ukd1** [comment]
You should checkout https://github.com/rainforestapp/yubikey as well - I got it nearly all the way but has issues with open ssl ECB

**Comment by ChrisLundquist** [comment]
I only saw the one commit,https://github.com/rainforestapp/yubikey/commit/8843b526c60c2d39659ccb636fc50fa7ed11d8da which I pulled in.

**Comment by ukd1** [comment]
Yeah I squashed it. Missed that you'd done that - great!

**Comment by ukd1** [comment]
Closed by a7617fe5d8606bd21e0df470a9180c00d07a5d5d

**Comment by titanous** [comment]
Thanks guys!

@ukd1 will handle releasing the gem.
